src/flb_lib.c

```diff
@@ -803,7 +803,7 @@ int flb_lib_push(flb_ctx_t *ctx, int ffd, const void *data, size_t len)
 /* Emulate some data from the response */
 int flb_lib_response(flb_ctx_t *ctx, int ffd, int status, const void *data, size_t len)
 {
-    int ret;
+    int ret = 0;
```

Comment:
Not related to my change but fixes this function returning an undefined value in some cases.